appveyor.yml

```diff
@@ -1,5 +1,5 @@
 version: 1.0.0.{build}
-image: Visual Studio 2017
+image: Visual Studio 2019 Preview
```

Comment:
This is just an enabler for C# 8.0, leaving `target_framework` compatibility untouched. Is my understanding correct?

Reply:
Yep. We don't need to touch the target frameworks. I have things compiling locally -- just waiting on AppVeyor to update now.

Reply:
Things look good on Travis.